pkg/tap/switch.go

```diff
@@ -159,7 +164,7 @@ func (e *Switch) txPkt(pkt *stack.PacketBuffer) error {
 		if err != nil {
 			return err
 		}
-		atomic.AddUint64(&e.Sent, uint64(pkt.Size()))
+		atomic.AddUint64(&e.Sent, uint64(size)) //#nosec:G115. Safely checked
```

Comment:
This `nosec` annotation is not needed on my system (I pushed this change to your branch)